recipes/arm_openmpi_install.rb

```diff
@@ -0,0 +1,46 @@
+
+# frozen_string_literal: true
+
+#
+# Cookbook Name:: aws-parallelcluster
+# Recipe:: arm_openmpi_install
+#
+# Copyright 2013-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance with the
+# License. A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file. This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES
+# OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+return if node['conditions']['ami_bootstrapped'] || !arm_instance?
+
+case node['platform_family']
+when 'amazon'
+  if node['platform_version'].to_i == 2
+    package 'openmpi-devel' do
+      retries 10
```

Comment:
why 10 retries?

Reply:
lol i don't have a good answer

Reply:
we have 10 retries for package installations randomly scattered throughout the recipes. agreed it's definitely not needed for these.

Reply:
`retries 10` is used when a list is passed to the package manager
`retries 3` is used when a single package is required to be installed

Reply:
ah i see.